code/src/App.js

```diff
@@ -1,9 +1,39 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+import { BrowserRouter, Switch, Route, Redirect } from "react-router-dom";
+
+import { API_KEY, MOVIES_URL } from "./utils/urls";
+
+import Loading from "./components/Loading";
+import { MovieList } from "./components/MovieList";
+import { MovieDetails } from "./components/MovieDetails";
+import NotFound from "./components/NotFound";
 
 export const App = () => {
+  const [movies, setMovies] = useState([]);
+  const [loading, setLoading] = useState(true)
+
+  useEffect(() => {
```

Comment:
If you want the loading component to showing just add this :) its not needed if you dont want that for this project

.then((res) => res.json())
      .then((json) => {
        setTimeout(() => setLoading(false), 2000);
        setMovies(json.results);
      });

Reply:
Thank you Jessica! Tried adding that now, but it still doesn't show so probably there is something more missing. Will have to look into that. :)